packages/engine-test-utils/CHANGELOG.md

```diff
@@ -12243,7 +12243,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Bug Fixes
 
-- handle single hiearchy properly ([3e822a6](https://github.com/dendronhq/dendron/commit/3e822a6d659c5f4da6bd3ddad9bf1d93a5c353e3))
+- handle single hierarchy properly ([3e822a6](https://github.com/dendronhq/dendron/commit/3e822a6d659c5f4da6bd3ddad9bf1d93a5c353e3))
```

Comment:
I think the changelog should stay as-is, it reflects what the release notes were at the time.

Reply:
I think the purpose of "change log" is not only to show the dev progress (when we have it in git commit message) but also to search? And when we search, ideally we won't write a wrong spelling?
Maybe you want to keep it for the git blame or some other reason. I don't know which is better, could you reconsider about this? And we do whatever you decided.

Reply:
Makes sense, we can keep it in.